spec/models/solidus_bolt/payment_method_spec.rb

```diff
@@ -18,4 +18,29 @@
       expect(described_class.new.partial_name).to eq 'bolt'
     end
   end
+
+  describe '#try_void' do
+    let(:payment) { create(:payment) }
+
+    context 'when the payment cannot be voided' do
```

Comment:
the context is when it CANNOT be voided, but the logic in the test is for when it CAN